woocommerce-custom-product-tabs-lite.php

```diff
@@ -263,8 +263,9 @@ public function product_write_panel() {
 	 */
 	public function product_save_data( $post_id, $post ) {
 
-		$tab_title   = sanitize_text_field( $_POST['_wc_custom_product_tabs_lite_tab_title'] );
+		// NOTE: custom HTML input is expected to be saved - this will be converted to a serialized array at the end of this method
```

Comment:
```suggestion
		// NOTE: custom HTML input is expected to be saved - this will be converted to a serialized array at the end of this method, and escaped with wp_kses_post() when output
```